yarn.lock

```diff
@@ -3475,6 +3475,16 @@
     tiny-warning "^1.0.3"
     tslib "^2.3.0"
 
+"@reach/router@^1.3.4":
```

Comment:
Did you happen to commit this file by mistake by any chance?

I saw you had modified `package.json` [before](https://github.com/vtex-sites/gatsby.store/pull/34/commits/37aed14e446f71c9c3ae959886bb887d23349184) but it's not here now so I'm not sure this file is needed.

Reply:
Oh yeah! My bad! I'll remove it! Thanks!